src/main/java/frc/robot/robot_manager/RobotManager.java

```diff
@@ -155,6 +156,12 @@ protected void afterTransition(RobotState newState) {
         intake.setState(IntakeState.INTAKING);
         queuer.setState(QueuerState.INTAKING);
       }
+      case INTAKE_ASSIST -> {
+        arm.setState(ArmState.IDLE);
+        shooter.setState(ShooterState.IDLE_STOPPED);
+        intake.setState(IntakeState.INTAKING);
+        queuer.setState(QueuerState.INTAKING);
+      }
```

Comment:
right now this is duplicating the actions in the `INTAKING` state. but i believe that's just because we don't tell swerve to enter the intake assist state here. need to add that in